.travis.yml

```diff
@@ -1,14 +1,26 @@
 language: c
-sudo: required
-dist: trusty
-group: edge
+
+matrix:
+  include:
+    - os: linux
+      sudo: required
+      dist: trusty
+      group: edge
+    - os: osx
 
 before_install:
-    - sudo apt-get -qq update
-    - sudo apt-get install -y libmpfr-dev libgomp1 gcc
-    
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository -y ppa:adrozdoff/cmake        ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update                               ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -y cmake libmpfr-dev libgomp1 gcc; fi
+
 compiler:
     - gcc
 
-script: make test
+before_script:
+  - mkdir sleef.build
+  - cd sleef.build
+  - cmake -DCMAKE_BUILD_TYPE=Release ..
+
+script:
+  - make test-libm VERBOSE=1
```

Comment:
I think you can remove `VERBOSE=1` from here.

Also, add a `make install` after the testing to make sure that also testing ends correctly.